debian/linglong-bin.postinst

```diff
@@ -1,98 +1,101 @@
-#!/bin/sh
+#!/bin/bash
 
 # SPDX-FileCopyrightText: 2023 UnionTech Software Technology Co., Ltd.
-#
 # SPDX-License-Identifier: LGPL-3.0-or-later
 
+log_file="/var/log/linglong-bin.log"  # Log file path
```

Comment:
Output (stdout and stderr) of debian maintainer scripts has been logged to /var/log/apt/term.log.